locale/en/knowledge/javascript-conventions/what-is-json.md

```diff
@@ -67,38 +73,42 @@ format. There are several gotchas that can produce invalid JSON as well.
 
 These are all examples of valid JSON.
 
-    {"name":"John Doe","age":32,"title":"Vice President of JavaScript"}
+```js
+{"name":"John Doe","age":32,"title":"Vice President of JavaScript"}
 
-    ["one", "two", "three"]
+["one", "two", "three"]
 
-    // nesting valid values is okay
-    {"names": ["John Doe", "Jane Doe"] }
+// nesting valid values is okay
+{"names": ["John Doe", "Jane Doe"] }
      
-    [ { "name": "John Doe"}, {"name": "Jane Doe"} ]
+[ { "name": "John Doe"}, {"name": "Jane Doe"} ]
 
-    {} // empty hash
+{} // empty hash
 
-    [] // empty list
+[] // empty list
 
-    null
+null
 
-    { "key": "\uFDD0" } // unicode escape codes
+{ "key": "\uFDD0" } // unicode escape codes
+```
 
 These are all examples of bad JSON formatting.
 
-    { name: "John Doe", 'age': 32 } // name and age should be in double quotes
+```js
```

Comment:
What about changing to 'json' instead of 'js' here? Because this is a pure 'json' formation :)

Reply:
Thanks for pointing out these changes. Might have slipped from my mind :)